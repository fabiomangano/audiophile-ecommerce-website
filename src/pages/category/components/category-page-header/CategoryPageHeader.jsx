import React from 'react';
import PropTypes from 'prop-types';
import style from './CategoryPageHeader.module.scss';

const CategoryPageHeader = ({ category }) => {
  return (
    <div className={style.container}>
      <h2>{category}</h2>
    </div>
  );
};

CategoryPageHeader.propTypes = {
  category: PropTypes.string.isRequired,
};

export default CategoryPageHeader;
