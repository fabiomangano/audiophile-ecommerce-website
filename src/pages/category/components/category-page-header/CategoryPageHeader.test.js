import React from 'react';
import { create } from 'react-test-renderer';
import { shallow } from 'enzyme';
import CategoryPageHeader from './CategoryPageHeader';

describe('CategoryPageHeader', () => {
  it('should renders correctly', () => {
    const tree = create(<CategoryPageHeader category="some-category" />);
    expect(tree.toJSON()).toMatchSnapshot();
  });

  it('should pass category prop correctly', () => {
    const wrapper = shallow(<CategoryPageHeader category="some-category" />);
    expect(wrapper.find('h2').text()).toBe('some-category');
  });
});
