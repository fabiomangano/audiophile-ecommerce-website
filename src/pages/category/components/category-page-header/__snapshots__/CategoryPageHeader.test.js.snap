// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryPageHeader should renders correctly 1`] = `
<div
  className="container"
>
  <h2>
    some-category
  </h2>
</div>
`;
